templates/launch.html

```diff
@@ -303,8 +374,19 @@ <h3>${survey_type} Survey Metadata</h3>
                         document.querySelector("#survey_metadata").innerHTML = "No metadata required for this survey";
                     }
 
-                    document.querySelector("#submit-btn").disabled = false;
-                    document.querySelector("#flush-btn").disabled = false;
+                    let launchPattern = document.querySelector("#launch_pattern").value
+
+                    if (launchPattern === "v1") {
+                        document.querySelector("#submit-btn-v1").style.display = 'inline-block';
+                    } else {
+                        document.querySelector("#submit-btn-v1").style.display = 'none';
+                    }
+                    if (launchPattern === "v2") {
+                        document.querySelector("#submit-btn-v2").style.display = 'inline-block';
+                    } else {
+                        document.querySelector("#submit-btn-v2").style.display = 'none';
+                    }
+                    document.querySelector("#flush-btn").style.display = 'inline-block';
```

Comment:
Cleaner if we make it one `if-else if-else`?:
```suggestion
                    if (launchPattern === "v1") {
                        document.querySelector("#submit-btn-v1").style.display = 'inline-block';
                        document.querySelector("#submit-btn-v2").style.display = 'none';
                    }
                    else if (launchPattern === "v2") {
                        document.querySelector("#submit-btn-v1").style.display = 'none';
                        document.querySelector("#submit-btn-v2").style.display = 'inline-block';
                    }
                    else {
                        document.querySelector("#submit-btn-v2").style.display = 'none';
                        document.querySelector("#submit-btn-v2").style.display = 'none';
                    }
                    document.querySelector("#flush-btn").style.display = 'inline-block';
```

Reply:
I think most of this should be removed. I would just create a function to toggle the button after loading from schema name or schema url. i.e


```js
    displayLaunchButton = () => {
        let launchPattern = document.querySelector("#launch_pattern").value
        document.querySelector(`#submit-btn-${launchPattern}`).style.display = 'inline-block';
        document.querySelector("#flush-btn").style.display = 'inline-block';
    }
```

Then just call this at the end of `loadMetadataForSchemaName()` and `loadMetadataForSchemaUrl`

Reply:
Added a function now 👍